packages/react/src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -21,38 +25,51 @@ export interface MenuVariables {
   primaryUnderlinedBorderColor: string
 
   disabledColor: string
-
   lineHeightBase: string
-
   horizontalPadding: string
+
+  verticalMenuBackgroundColor: string
```

Comment:
`NIT: ~~`verticalMenuBackgroundColor `~~ -> `verticalBackgroundColor ` since we're already in the context of the menu

same for `verticalMenuItemPadding`, `verticalMenuBoxShadow`, `menuDividerHeight `, `menuBorderWidth `